.github/workflows/tests.yml

```diff
@@ -38,6 +67,10 @@ jobs:
             /bin/bash uninstall.sh -f >/dev/null
           fi
 
+      - name: Set up WSL sudo
+        if: runner.os == 'windows'
+        run: echo "$USER ALL=(ALL) NOPASSWD:ALL" | sudo tee "/etc/sudoers.d/$USER"
```

Comment:
You are trying to use `sudo` to allow you doing `sudo` ;-)

Reply:
lol yup, doh.